src/assets/scripts/client/aircraft/AircraftController.js

```diff
@@ -921,4 +920,17 @@ export default class AircraftController {
             this.aircraft_remove(aircraftModel);
         }
     }
+
+    /**
+     * Respond to user changing the tower control (autotower) settings
+     *
+     * @for AircraftController
+     * @private
+     */
+    _onTowerControllerChange() {
+        const isAutoTower = GameController.getGameOption(GAME_OPTION_NAMES.TOWER_CONTROLLER) === 'SYSTEM';
```

Comment:
with my suggestion above, this line goes away completely

Reply:
Again, fetch once and reuse